tests/Hosting.Services.Web.UnitTests/Middlewares/UserIdentity/EmailBasedUserIdentityProviderTests.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Web.UnitTests
+{
+	[TestClass]
+	public class EmailBasedUserIdentityProviderTests
+	{
+		[TestMethod]
+		public async Task TryWriteBytes_ReturnFalseIfNoEmail()
```

Comment:
Follow the Arrange-Act-Assert (AAA) unit testing pattern for clarity. See <https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/>

The method names should be in the format of `TryWriteBytesAsync_IfNoEmail_ReturnsFalse` etc. It's also a good idea to skip a line between each of the assert, act, and assert blocks.

Reply:
Was following the pattern in IpBasedUserIdentityProviderTests.cs, will change to AAA pattern now.